src/qt/anybutton.cpp

```diff
@@ -33,9 +34,74 @@ class wxQtPushButton : public wxQtEventSignalHandler< QPushButton, wxAnyButton >
 wxQtPushButton::wxQtPushButton(wxWindow *parent, wxAnyButton *handler)
     : wxQtEventSignalHandler< QPushButton, wxAnyButton >( parent, handler )
 {
+    installEventFilter( this );
     connect(this, &QPushButton::clicked, this, &wxQtPushButton::clicked);
 }
 
+bool wxQtPushButton::eventFilter(QObject *watched, QEvent *event)
+{
+    if( watched != this )
```

Comment:
Minor, but please use `if (` style, i.e. put spaces before `(` (here and everywhere).